src/braket/circuits/circuit.py

```diff
@@ -82,6 +82,7 @@ class Circuit:
     """
 
     _ALL_QUBITS = "ALL"  # Flag to indicate all qubits in _qubit_observable_mapping
+    default_diagram_builder = BoxDrawingCircuitDiagram
```

Comment:
We may not want to change the default until we've confirmed that there's no breaking changes when running our notebooks.